test/common/markdown/markdown.tests.ps1

```diff
@@ -102,15 +102,18 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
 
         $mdIssuesPath | should exist
 
+        [string] $markdownErrors = ""
         Get-Content -Path $mdIssuesPath | ForEach-Object -Process {
             if ([string]::IsNullOrEmpty($_) -eq $false -and $_ -ne '--EMPTY--')
             {
                 Write-Warning -Message $_
```

Comment:
I guess this line can be removed now. It supposes to provide detailed errors but looks like warnings are not captured by AppVeyor.